app/Http/Controllers/SubpageController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Subpage;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class SubpageController extends Controller
+{
+    /**
+     * Show a subpage from an id.
+     *
+     * @param int $id
+     * @return View
+     */
+    public function show($id) : View
```

Comment:
@abm-devmonk-purple We should convert this and any routes going forward to use route model binding.

Reply:
Done in e6ac5b1d5fd457b5616405fefd32d856b9624c3a.